packages/client/hmi-client/src/components/widgets/share-project/tera-share-project.vue

```diff
@@ -69,6 +69,7 @@ import Button from 'primevue/button';
 import { getUsers } from '@/services/user';
 import type { PermissionRelationships, PermissionUser, Project } from '@/types/Types';
 import { useProjects } from '@/composables/project';
+import { cloneDeep } from 'lodash';
```

Comment:
We want to move away from `cloneDeep` and use the Web API [structuredClone()](https://developer.mozilla.org/en-US/docs/Web/API/structuredClone) now that this is supported by all browser.